packages/amplify-provider-awscloudformation/src/aws-utils/aws-cfn.js

```diff
@@ -153,7 +155,8 @@ class CloudFormation {
   }
 
   readStackEvents(stackName) {
-    this.pollForEvents = setInterval(() => this.addToPollQueue(stackName, 3), CFN_POLL_TIME);
+    ++this.readStackEventsCalls;
+    this.pollForEvents = setInterval(() => this.addToPollQueue(stackName, 3), CFN_POLL_TIME + (this.readStackEventsCalls * CFN_POLL_TIME_BACKOFF));
```

Comment:
Should be something like
```suggestion
    this.pollForEvents = setInterval(() => this.addToPollQueue(stackName, 3), ((2^ this.readStackEventsCalls)* CFN_POLL_TIME));
    this.readStackEventsCalls++;
```

Above increment can be removed to start with ```CFN_POLL_TIME```